### PR TITLE
Made part and part_of properties facetable.

### PR DESCRIPTION
**Pull request by DeonFranklin**
Prior to this commit:
Part and part_of properties were not faceted

With this commit:
The catalog_controller, dog_biscuits, and _attribute_rows were updated to facet the properties.

Note:
This does not include collections as collections require additional work.

Ref# 280

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -98,10 +98,11 @@ def self.uploaded_field
     config.add_facet_field solr_name("subject", :facetable), limit: 5
     config.add_facet_field solr_name("language", :facetable), limit: 5
     config.add_facet_field solr_name("based_near_label", :facetable), limit: 5
+    config.add_facet_field solr_name("part", :facetable), limit: 5, label: 'Part'
     config.add_facet_field solr_name("part_of", :facetable), limit: 5
     # config.add_facet_field solr_name("file_format", :facetable), limit: 5
     # config.add_facet_field solr_name("contributor", :facetable), label: "Contributor", limit: 5
-    config.add_facet_field solr_name('member_of_collections', :symbol), limit: 5, label: 'Collections'
+    config.add_facet_field solr_name('member_of_collections', :symbol), limit: 5, label: 'Collection'
```
Comment (jeremyf) [inline app/controllers/catalog_controller.rb:105]: Do you know why this changed?

**Review by kirkkwang — APPROVED** [review]
Looks good!

**Review by jeremyf — CHANGES_REQUESTED** [review]
Before you merge, I'd like to know about changing "Collections" to "Collection"

File: app/controllers/catalog_controller.rb (hunk 1 above)
Comment (DeonFranklin) [reply app/controllers/catalog_controller.rb]: It should not have been changed and was fixed.

**Review by jeremyf — APPROVED** [review]

**Pull request merged**